packages/jerboa/jerboa.0.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+name: "jerboa"
+version: "0.1"
+synopsis: "Jerboa is a minimalistic web framework for everyone"
+description: """
+Jerboa is web framework, which is:
+- Minimalistic: It will give you the building blocks, but nothing more
+- Flexible: It let's you be as flexible as you want to be
+- Easy to use: You only need to understand the simple building blocks to use it
+"""
+maintainer: "Robert Toth <kkdstryker@gmail.com>"
+authors: ["Robert Toth"]
+homepage: "https://github.com/StrykerKKD/Jerboa"
+bug-reports: "https://github.com/StrykerKKD/Jerboa/issues"
+dev-repo: "git+https://github.com/StrykerKKD/Jerboa.git"
+doc: "https://strykerkkd.github.io/Jerboa/"
+license: "MIT"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name] {with-doc}
+]
+run-test: [
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

Comment:
This doesn't work. It's either `{with-test}` inside the `build` phase or just bare commands inside the `run-test` phase.